web-marketplace/src/pages/Post/Post.Header.tsx

```diff
@@ -134,8 +143,10 @@ export const PostHeader = ({
                   setShareSuccessBanner(true);
                 }}
               />
-              {/* <SharePrivateMenuItem />
-              <DeleteMenuItem /> */}
+              {!publicPost && (
+                <SharePrivateMenuItem onClick={sharePrivateLink} />
```

Comment:
TODO add SharePrivateMenuItem to data stream PostCard once https://github.com/regen-network/regen-web/pull/2390 is merged